### PR TITLE
 Issue #634: satelite projects should use latest checkstyle as soon at is appears in maven repo

### PR DESCRIPTION
**Pull request by Rahulkhinchi03**
Partially Fixes #634

### CODE REVIEW

File: .github/workflows/bump_project_version.yml [hunk 1]
```diff
@@ -0,0 +1,22 @@
+#############################################################################
+# GitHub Action to Bump Project Version.
+#
+# Workflow starts every day at the end of the day.
+#
+#############################################################################
+name: "Bump Project Version"
```
Comment (romani) [inline .github/workflows/bump_project_version.yml:7]: fix name and apply all comment from https://github.com/checkstyle/contribution/pull/635#pullrequestreview-1079599409

**Review by romani — CHANGES_REQUESTED** [review]
itms:

File: .github/workflows/bump_project_version.yml (hunk 1 above)
Comment (Rahulkhinchi03) [reply .github/workflows/bump_project_version.yml]: Changes Done

**Review by romani — APPROVED** [review]
ok to merge

**Pull request merged**